src/utilities/gql.ts

```diff
@@ -1405,6 +1406,14 @@ const gql = {
     }
   `,
 
+  SIMULATION_DATASET_ID: `#graphql
+  query GetSimulationDatasetId($datasetId: Int!) {
```

Comment:
do you mind adding an extra tab for before the query block, please?

```
SIMULATION_DATASET_ID: `#graphql
  query GetSimulationDatasetId($datasetId: Int!) {
```

Reply:
done